gloo/docs/gloo_routing/virtual_services/security/ldap/vs-echo-no-auth.yaml

```diff
@@ -13,6 +13,8 @@ spec:
         prefix: /echo
       routeAction:
         single:
-          upstream:
-            name: default-http-echo-5678
-            namespace: gloo-system
+          kube:
+            ref:
+              name: http-echo
+              namespace: default
+            port: 5678
```

Comment:
OOC, why the change from the discovered upstream to a direct kube service ref?

Reply:
I thought since we just created a kube service and it reads:
> Now we can create a Virtual Service that routes any requests with the `/echo` prefix to the `http-echo` service

that it would be more appropriate to use the kube service destination. In the previous tutorials we use static upstreams, so it is more intuitive to use the upstream destination. This way we also showcase both features. :)